next/components/forms/simple-components/Banner.tsx

```diff
@@ -76,7 +76,7 @@ const Banner = ({
         </div>
       </div>
       <div className="my-auto flex w-full sm:w-1/2">
-        <Image src={image} />
+        <Image src={image} alt="" />
```

Comment:
we should update the Banner api here - force it to pass alt from props and use that in here

Reply:
Good point, fixed